src/views/services/form/ServiceForm.tsx

```
@@ -0,0 +1,217 @@
+import { useNavigate } from 'react-router';
+
+import { ServiceModel } from '@kubevirt-ui/kubevirt-api/console';
+import { IoK8sApiCoreV1Service } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { k8sCreate, k8sUpdate } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Form,
+  FormGroup,
+  PageSection,
+  TextArea,
+  TextInput,
+  Content,
```

ESLint

[error] Expected "Content" to come before "TextInput"

(perfectionist/sort-named-imports)